src/libsystemd/libsystemd.sym

```diff
@@ -1046,6 +1046,7 @@ global:
         sd_varlink_take_fd;
         sd_varlink_unref;
         sd_varlink_wait;
+        sd_device_get_driver_subsystem;
```

Comment:
hmm, the various sd_device_get_xyz() calls all have a man page. Any chance we can get this one added there, right away too?